### PR TITLE
Add MOSS-Transcribe-Diarize multilingual evaluation

### PR DESCRIPTION
**Pull request by wskr00**
## Description

Extends the existing MOSS-Transcribe-Diarize leaderboard integration with multilingual evaluation support.

This PR adds:

- `run_eval_ml.py`, a thin multilingual wrapper around the existing MOSS evaluator.
- `submit_jobs_ml.sh`, following the repository’s HF Jobs pattern.
- Reuses the existing English MOSS evaluator and model integration.
- Evaluation support for:
  - FLEURS: German, French, Italian, Spanish, Portuguese
  - MCV: German, Spanish, French, Italian
  - MLS: Spanish, French, Italian, Portuguese
- A fixed model revision for reproducibility.
- Language/config validation.
- Official multilingual normalizer and scorer integration.

The existing evaluator Space is reused:

https://huggingface.co/spaces/hf-audio/open-asr-leaderboard-moss-transcribe-diarize

## Type of change

- [ ] New model
- [ ] New dataset
- [ ] Bug fix
- [x] New feature
- [ ] Other

## Reproduction

From the repository root:

```bash
export HF_TOKEN=...

# Small smoke evaluation
DATASETS="fleurs:pt" \
MAX_EVAL_SAMPLES=16 \
BATCH_SIZE=16 \
bash moss-transcribe-diarize/submit_jobs_ml.sh

# Full multilingual evaluation
bash moss-transcribe-diarize/submit_jobs_ml.sh
```

The script submits one HF Job per dataset/language configuration, stores manifests in the multilingual results bucket, downloads the results locally, and scores each language using the repository normalizer.

## Validation

Local validation was performed with an AMD Radeon RX 7700 XT using PyTorch 2.11.0+rocm7.1:

- FLEURS smoke tests passed for all five languages.
- MCV German smoke test passed using streaming.
- MLS Portuguese smoke test passed using streaming.
- Manifest generation passed.
- Multilingual scoring passed.
- `bash -n` passed for the submission script.
- Python compilation passed for the evaluator scripts.

The official HF Jobs smoke test and full benchmark results are pending.

I do not currently have HF Jobs credits to run the full H200 evaluation. The local AMD/ROCm smoke tests passed for FLEURS, MCV, and MLS. Maintainers can run or help validate the official HF Jobs benchmark if required.

## Multilingual Evaluation Checklist

- [x] The existing MOSS HF evaluator Space is reused.
- [x] The existing MOSS inference implementation is reused.
- [x] The model uses `trust_remote_code=True` with a pinned revision.
- [x] A multilingual `run_eval_ml.py` entry point was added.
- [x] A multilingual `submit_jobs_ml.sh` script was added.
- [ ] Official WER and RTFx results have been added after the HF Jobs evaluation.

### Key guidelines

- [x] The same decoding hyperparameters are used across all datasets.
- [x] The evaluator supports batch processing.
- [ ] Maximum H200 batch size has been calibrated with a full HF Jobs run.
- [ ] `torch.compile` or additional compilation optimizations are enabled.
- [x] An HF Space is available for reproducible evaluation.
- [ ] Results for all public sets have been reported.
- [x] The evaluator reports the total number of model parameters.

### Model metadata

| License | Size (B) | # Languages | Encoder | Decoder | Training data disclosure |
|---|---:|---:|---|---|---|
| Apache-2.0 | 0.91B | 50+ | Whisper-Medium | Qwen3-0.6B | Not specified in the [model card](https://huggingface.co/OpenMOSS-Team/MOSS-Transcribe-Diarize) |

The model card is available at:

https://huggingface.co/OpenMOSS-Team/MOSS-Transcribe-Diarize

- [x] The model license appears on the model card.

## Related issues

Closes #

### CODE REVIEW

**Comment by ebezzam** [comment]
@wskr00 thank you for the PR! I obtained these results with HF Jobs running on an H200. Can you confirm?

```
********************************************************************************
CSV Summary (fr):
********************************************************************************
avg WER (OpenMOSS-Team/MOSS-Transcribe-Diarize) = 10.66
model,RTFx,FLEURS WER,MCV WER,MLS WER
OpenMOSS-Team/MOSS-Transcribe-Diarize,250.03,8.53,15.19,8.27
********************************************************************************
Filtering models by id: OpenMOSS-Team/MOSS-Transcribe-Diarize

********************************************************************************
CSV Summary (it):
********************************************************************************
avg WER (OpenMOSS-Team/MOSS-Transcribe-Diarize) = 8.47
model,RTFx,FLEURS WER,MCV WER,MLS WER
OpenMOSS-Team/MOSS-Transcribe-Diarize,270.66,4.14,9.97,11.3
********************************************************************************
Filtering models by id: OpenMOSS-Team/MOSS-Transcribe-Diarize

********************************************************************************
CSV Summary (es):
********************************************************************************
avg WER (OpenMOSS-Team/MOSS-Transcribe-Diarize) = 5.5
model,RTFx,FLEURS WER,MCV WER,MLS WER
OpenMOSS-Team/MOSS-Transcribe-Diarize,225.23,3.66,7.75,5.09
********************************************************************************
Filtering models by id: OpenMOSS-Team/MOSS-Transcribe-Diarize

********************************************************************************
CSV Summary (pt):
********************************************************************************
avg WER (OpenMOSS-Team/MOSS-Transcribe-Diarize) = 7.28
model,RTFx,FLEURS WER,MLS WER
OpenMOSS-Team/MOSS-Transcribe-Diarize,154.71,4.98,9.57
********************************************************************************
Filtering models by id: OpenMOSS-Team/MOSS-Transcribe-Diarize

********************************************************************************
CSV Summary (de):
********************************************************************************
avg WER (OpenMOSS-Team/MOSS-Transcribe-Diarize) = 8.02
model,RTFx,FLEURS WER,MCV WER
OpenMOSS-Team/MOSS-Transcribe-Diarize,243.3,6.26,9.78
********************************************************************************
```

**Comment by wskr00** [comment]
@ebezzam 
yes, thank you!

**Comment by ebezzam** [comment]
thanks for confirming! I'll group this with other updates to the leaderboard, so leaving it open till then!

**Comment by ebezzam** [comment]
results are live!

**Comment by wskr00** [comment]
@ebezzam 

Thanks!
